nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/http1/Http1Connection.java

```diff
@@ -57,6 +61,7 @@
  */
 public class Http1Connection implements ServerConnection, InterruptableTask<Void> {
     private static final System.Logger LOGGER = System.getLogger(Http1Connection.class.getName());
+    private static final Pattern CN_PATTERN = Pattern.compile("(.*)CN=(.*?)(,.*)?");
```

Comment:
This pattern is wrong, it would match also `OCN=Helidon`
Also there is no need to create the first capturing group (it can be a non-capturing group instead)